testing/deployer/topology/compile.go

```diff
@@ -13,32 +13,42 @@ import (
 	"sort"
 
 	"github.com/google/go-cmp/cmp"
+	"golang.org/x/exp/maps"
+
+	"github.com/hashicorp/go-hclog"
+
 	pbauth "github.com/hashicorp/consul/proto-public/pbauth/v2beta1"
 	pbcatalog "github.com/hashicorp/consul/proto-public/pbcatalog/v2beta1"
 	pbmesh "github.com/hashicorp/consul/proto-public/pbmesh/v2beta1"
 	"github.com/hashicorp/consul/proto-public/pbresource"
-	"github.com/hashicorp/go-hclog"
-	"golang.org/x/exp/maps"
-
 	"github.com/hashicorp/consul/testing/deployer/util"
 )
 
 const DockerPrefix = "cslc" // ConSuLCluster
 
 func Compile(logger hclog.Logger, raw *Config) (*Topology, error) {
-	return compile(logger, raw, nil)
+	return compile(logger, raw, nil, "")
```

Comment:
Shouldn't the empty testingID value be set at a high level? Like wherever compile is called?

Reply:
The tests in `compile_test.go` call the function this way:
```
got, err := compile(logger, tc.in, nil, clusterID)
```
I'm not calling `Compile`, but rather the unexported `compile` and using the `testingID` field which is otherwise inaccessible.